src/app/components/site-info/index.js

```diff
@@ -1,62 +1,78 @@
+	return (
+		<div className="wppbh-app-site-info nfd-bg-[#212936] nfd-w-full nfd-py-6 nfd-px-8 nfd-mb-8 nfd-border nfd-border-line nfd-rounded-lg">
+			<div className="nfd-flex nfd-justify-between nfd-items-center nfd-flex-wrap nfd-gap-4">
+				<div className="nfd-w-max nfd-flex nfd-flex-col nfd-gap-1.5">
+					<h3 className="nfd-text-white nfd-text-2xl nfd-font-semibold">
+						{ title }
+					</h3>
+					<div className="nfd-flex nfd-items-center nfd-gap-3 nfd-font-medium">
+						<div className="nfd-flex nfd-items-center nfd-gap-1">
+							{ renderPadLock() }
+							<span className="nfd-text-white nfd-text-tiny">
+								{ siteDomain }
+							</span>
+						</div>
+					</div>
+				</div>
 
-                <div className="nfd-w-max nfd-flex nfd-flex-col nfd-gap-1.5">
-                    <h3 className="nfd-text-white nfd-text-2xl nfd-font-semibold">{title}</h3>
-                    <div className="nfd-flex nfd-items-center nfd-gap-3 nfd-font-medium">
-                        <div className="nfd-flex nfd-items-center nfd-gap-1">
-                            {renderPadLock()}
-                            <span className="nfd-text-white nfd-text-tiny">{siteDomain}</span>
-                        </div>
-                    </div>
-                </div>
-
-                <div className="nfd-w-max nfd-flex nfd-items-center nfd-flex-wrap nfd-gap-3">
-                    <Button 
-                        as="a"
-                        href={addUtmParams(getPlatformPathUrl("hosting/details", "app/#/sites"))}
-                        target="_blank"
-                        variant="primary" 
-                        className="nfd-bg-[#383F4A] nfd-text-tiny nfd-w-full min-[400px]:nfd-w-auto">
-                        <BluehostIcon />
-                        Bluehost Account
-                    </Button>
-                    <Button 
-                        as="a" 
-                        href={(isWooCommerce && isStore) ? `${url}/shop` : url}
-                        target="_blank" 
-                        variant="primary" 
-                        className="nfd-bg-white nfd-text-[#212936] nfd-text-tiny nfd-w-full min-[400px]:nfd-w-auto"
-                    >
-                        <WordPressIcon />
-                        {(isWooCommerce && isStore) ? 'View Store' : 'View Site'}
-                    </Button>
-                </div>
-                
-            </div>
-        </div>
-    );
-}
\ No newline at end of file
+				<div className="nfd-w-max nfd-flex nfd-items-center nfd-flex-wrap nfd-gap-3">
+					<Button
+						as="a"
+						href={ addUtmParams(
+							getPlatformPathUrl(
+								'hosting/details',
+								'app/#/sites'
+							)
+						) }
+						target="_blank"
+						variant="primary"
+						className="nfd-bg-[#383F4A] nfd-text-tiny nfd-w-full min-[400px]:nfd-w-auto"
+					>
+						<BluehostIcon />
+						Bluehost Account
```

Comment:
:1234: Need to make this translatable

Reply:
Addressed in this commit: https://github.com/bluehost/bluehost-wordpress-plugin/pull/820/commits/f2189f1882d8bc9b38edf951a32c2bcc2d63133f